esp32_jpeg_enc/main/jpeg_decoder.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+
+#include <esp_log.h>
+#include <esp_err.h>
+
+#include "esp_audio_mem.h"
+#include "smtp_client.h"
+
+static const char *TAG="jpeg_decoding";
+
+
+extern const uint8_t raw_image_start[] asm("_binary_esp_img_start");
+extern const uint8_t raw_image_end[]   asm("_binary_esp_img_end");
+
+
+/*extern const uint8_t raw_image_start[] asm("_binary_raw_image_start");
+extern const uint8_t raw_image_end[]   asm("_binary_raw_image_end");*/
+
+int decode_jpeg(unsigned char *source_buf,
+                     unsigned long *source_buf_sz,
+                     unsigned char *out_buf
+                     );
+
+unsigned long in_buf_sz = 96 * 96 * 4;
```

Comment:
Number 96x96 doesn't make sense.

Use something to denote how much KB. e.g `10 * 1024` meaning 10KB